lib/src/features/admin/abilities/data/abilities_repository.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:convert';
+
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../../../utilities/base_repository.dart';
+import '../../../authentication/data/auth_repository.dart';
+import '../../../group_management/application/has_group_controller.dart';
+import '../domain/ability.dart';
+
```

Comment:
hier dann auf jeden Fall auch dran denken das von Name auf Id umzustellen

Reply:
Also weiter unten im code xD

Reply:
Muss ich überall in client und server